pkg/agent/multicluster/mc_route_controller_test.go

```diff
@@ -555,8 +563,8 @@ func TestEnqueueClusterInfoImport(t *testing.T) {
 		agent.WireGuardConfig{},
 		mockInterface,
 		"wireGuard",
+		&wgtest.MockInterface{},
```

Comment:
why doesn't it initialize via `wgtest.NewMockInterface(ctrl)`? Without a nil controller, code would panic when it's called.

Reply:
These cases doesn't cover WireGuard related codes, but you are right, I will update them in case there is any case changes in the future.

Reply:
It could pass `nil` if it's not supposed to be used. Passing an invalid instance could be confusing.

Reply:
Make sense, change it to nil, thanks.